argus/backend/util/config.py

```diff
@@ -12,6 +12,7 @@ class Config:
     CONFIG_PATHS = [
         Path("./config/argus_web.yaml"),
         Path("argus_web.yaml"),
+        Path("../argus_web.yaml"),
```

Comment:
Do we need this for this PR or is this something QoL related?

Reply:
QoL only

Reply:
removed